src/conda/env_synthetic_data.yml

```diff
@@ -0,0 +1,17 @@
+# Conda environment for synthetic data generation
```

Comment:
@wrongkindofdoctor - in your testing, do you feel that the MDTF synthetic data package is mature enough that it should go up on PyPi and anaconda.org to make installation easier.  I can do this if you think it's ready and would be helpful.

Reply:
I think that that the synthetic data package should be modified to be more flexible. It relies on hard-coded variables and metadata used by some of the current PODs, but this will be difficult to maintain as we add more PODs, and support for CMIP-style variables, to the CI. If we add an arparse routine to accept GFDL, CESM, or CMIP as the convention, consolidate the GFDL and NCAR synthetic scripts into one main script, and move the variables and metadata to jsons (or some other format), we can extend the variables by amending/adding spec files.